bin/process_h5ad.py

```diff
@@ -19,10 +22,25 @@ def h5ad_to_zarr(
     compute_embeddings=False,
     chunk_size=10,
     var_index=None,
+    batch_processing=False,
+    batch_size=10000,
 ):
 
     if not adata:
-        adata = sc.read(file)
+        if not batch_processing:
+            adata = sc.read(file)
+        else:
+            with h5py.File(file, "r") as f:
+                # Load everything but X and layers
```

Comment:
I was wondering if it was worth excluding things that are not used within the Vitessce/SpatialData implementation? For example, `uns`. Sometime people put weird things here that are very large. If the motivation for this batch processing is processing large files, then maybe drop those things?

Reply:
I've included `uns` here just to ensure that the same zarr is written using batch processing or not. But we can definitely remove it from both options as we haven't identified any case where anything in `uns` is used afterwards.
I'll leave it for this PR and we can make changes in dev